src/cli.zig

```diff
@@ -1180,7 +1180,27 @@ pub const Command = struct {
             RootCommandMatcher.case("init") => .InitCommand,
             RootCommandMatcher.case("build"), RootCommandMatcher.case("bun") => .BuildCommand,
             RootCommandMatcher.case("discord") => .DiscordCommand,
-            RootCommandMatcher.case("upgrade") => .UpgradeCommand,
+            RootCommandMatcher.case("upgrade") => brk: {
+                if (args_iter.buf.len > 2) {
```

Comment:
is this the best place to put this, can we do this check in upgrade_command.zig?

Reply:
Hey @paperdave, I believe this commit (https://github.com/oven-sh/bun/pull/7784/commits/5279915ba160472fbedea4fcd239138574527ecb) resolves this. Please let me know if not :)